Sources/JWTKit/JWK/JWK.swift

```diff
@@ -79,26 +107,63 @@ public struct JWK: Decodable {
     ///
     /// The `kid` value is a case-sensitive string.
     public var keyIdentifier: JWKIdentifier?
-    
+
+    // RSA keys
+    // Represented as the base64url encoding of the value’s unsigned big endian representation as an octet sequence.
     /// `n` Modulus.
     public var modulus: String?
-    
+
     /// `e` Exponent.
     public var exponent: String?
-    
+
     /// `d` Private exponent.
     public var privateExponent: String?
-    
+
+    // ECDSA keys
+    public var x: String?
```

Comment:
Are there more "technically correct" names than x and y? i.e., `d` in RSA is actually the "private exponent"

Reply:
It doesn't really have a good name as far as I can see. We can call it a `generatorPoint` and rename `Parameters` to `GeneratorPoint` as well as it's a point on the elliptic curve apparently :) How does that sound? But then I'll redo the storing in that way as well instead of x and y separate

Reply:
@tanner0101 I've implemented your other suggestions. Let me know what you think regarding the above!